src/test/java/fr/ans/psc/pscextract/ExtractionControllerTests.java

```diff
@@ -99,49 +99,34 @@ private void Clean() {
   @Test
   void singlePageExtractionAndResultConformityTest() throws Exception {
 
-    String responseFilename = "multiple-work-situations-result";
-    String responsePath = Objects.requireNonNull(Thread.currentThread().getContextClassLoader().getResource("wiremock/__files/" + responseFilename + ".txt")).getPath().substring(1);
-    byte[] expectedResponseBytes = Files.readAllBytes(Paths.get(responsePath));
-
     httpMockServer.stubFor(get("/v2/ps?page=0&size=1").willReturn(aResponse().withStatus(200).withHeader("Content-Type", "application/json").withBodyFile("multiple-work-situations.json")));
     httpMockServer.stubFor(get("/v2/ps?page=1&size=1").willReturn(aResponse().withStatus(410)));
 
     controller.generateExtract(null);
-
     await().until(controllerIsReady(controller));
 
     ResponseEntity<FileSystemResource> response = controller.getFile();
 
-    byte[] responseBytes = getBytesFromResponse(response);
-
-    String expected = new String(expectedResponseBytes, StandardCharsets.UTF_8);
-    String actual = new String(responseBytes, StandardCharsets.UTF_8);
-
```

Comment:
Two problems here : 
- the resource.getPath() method leaves an invalid character (<:>) at index 2 while running on Windows environment. Ref: https://stackoverflow.com/questions/43972777/java-nio-file-invalidpathexception-illegal-char-at-index-2
That's why I used to call substring(1), which isn't a clean solution => I changed it to a File based approach (see getContentAsString method below)
- EOL errors in expected results files : add a getNormalizedEOL() method to neutralize Windows EOL conversions.